clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/ubam/UbamMetadata.scala

```diff
@@ -46,7 +46,8 @@ case class UbamMetadata(
   ubamMd5: Option[Symbol] = None,
   ubamPath: Option[URI] = None,
   ubamSize: Option[Long] = None,
-  aggregatedBy: Option[AggregatedBy] = None,
+  aggregatedBy: Option[AggregatedBy] = None, // No longer used. Superseded by aggregationProject
```

Comment:
I think it's clearer to put `//` comments on a line by themselves, instead of as trailing comments. Can you move this to the previous line?

If this is no longer used does it need to be in the case class?